cluster/common.sh

```diff
@@ -322,15 +328,18 @@ function find-tar() {
 #   SERVER_BINARY_TAR
 #   KUBE_MANIFESTS_TAR
 function find-release-tars() {
+  # shellcheck disable=SC2034
   SERVER_BINARY_TAR=$(find-tar kubernetes-server-linux-amd64.tar.gz)
   if [[ "${NUM_WINDOWS_NODES}" -gt "0" ]]; then
+    # shellcheck disable=SC2034
```

Comment:
why?

Reply:
Same as above.

Reply:
https://github.com/kubernetes/kubernetes/pull/82454/files#r324942507

Reply:
update in comment